feature/experimental/backup_nhg/ate_tests/backup_nhg_test/backup_nhg_test.go

```diff
@@ -329,96 +264,52 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 
 }
 
-// configureBackupNextHopGroup creates and deletes the gribi nexthops, nexthop
-// groups, and prefixes for evaluating a single backup next hop forwarding
+// configureBackupNextHopGroup creates gribi nexthops, nexthop
+// groups, and prefixes for evaluating backup next hop forwarding
 // entry.
 func (a *testArgs) configureBackupNextHopGroup(t *testing.T, del bool) {
-	const (
-		// Next hop group ID that the dstPfx will forward to.
-		dstNHGID = 101
-		// Backup next hop group ID that the dstPfx will forward to.
-		dstBackupNHGID = 102
-
-		dutPort2ID, dutPort3ID = 10002, 10003
-	)
-
-	nh1 := fluent.NextHopEntry().WithNetworkInstance(*deviations.DefaultNetworkInstance).
-		WithIndex(dutPort2ID).WithIPAddress(atePort2.IPv4)
-	nh2 := fluent.NextHopEntry().WithNetworkInstance(*deviations.DefaultNetworkInstance).
-		WithIndex(dutPort3ID).WithIPAddress(atePort3.IPv4)
-
-	nhg := fluent.NextHopGroupEntry().WithNetworkInstance(*deviations.DefaultNetworkInstance).
-		WithID(dstNHGID).AddNextHop(dutPort2ID, 1).WithBackupNHG(dstBackupNHGID)
-	bnhg := fluent.NextHopGroupEntry().WithNetworkInstance(*deviations.DefaultNetworkInstance).
-		WithID(dstBackupNHGID).AddNextHop(dutPort3ID, 1)
-
-	pfx := fluent.IPv4Entry().WithNetworkInstance(*deviations.DefaultNetworkInstance).
-		WithPrefix(dstPfx).WithNextHopGroup(dstNHGID)
-
-	if del {
-		a.gribic.Modify().DeleteEntry(t, pfx, nhg, bnhg, nh2, nh1)
-	} else {
-		a.gribic.Modify().AddEntry(t, nh1, nh2, bnhg, nhg, pfx)
-	}
-	if err := awaitTimeout(a.ctx, a.gribic, t, time.Minute); err != nil {
-		t.Fatalf("Await got error for entries: %v", err)
-	}
+	t.Logf("Adding NH %d with atePort2 via gRIBI", NH1ID)
+	a.client.AddNH(t, NH1ID, atePort2.IPv4, *deviations.DefaultNetworkInstance, fluent.InstalledInFIB)
+	t.Logf("Adding NH %d with atePort3 and NHGs %d, %d via gRIBI", NH2ID, NH1ID, NH2ID)
+	a.client.AddNH(t, NH2ID, atePort3.IPv4, *deviations.DefaultNetworkInstance, fluent.InstalledInFIB)
+	a.client.AddNHG(t, NH1ID, map[uint64]uint64{NH1ID: 100}, *deviations.DefaultNetworkInstance, fluent.InstalledInFIB)
```

Comment:
Could you please add separate consts for NHG ids (just to increase readability; you can use the same values)

Reply:
Sure, have addressed this.